lib/src/wolt_modal_sheet_route.dart

```diff
@@ -1,13 +1,21 @@
 import 'package:flutter/material.dart';
 import 'package:wolt_modal_sheet/wolt_modal_sheet.dart';
 
+double _defaultModalTypeBreakPoint = 768.0;
```

Comment:
It would be still beneficial if we leave the user the choice to overwrite this value if needed. Making a default fallback value is fine, but I would not like to force the user upon a value. WDYT?